frontend/src/utils/constants.ts

```diff
@@ -2,6 +2,8 @@ import { StaticJsonRpcProvider } from 'src/utils/ethers';
 
 export const MAINNET_RPC_URL = String(process.env.MAINNET_RPC_URL);
 export const MAINNET_PROVIDER = new StaticJsonRpcProvider(MAINNET_RPC_URL);
+export const POLYGON_MAINNET_RPC_URL = `https://polygon-mainnet.infura.io/v3/${String(process.env.INFURA_ID)}`;
```

Comment:
Probably should make this a generic RPC URL like we do with `MAINNET_RPC_URL`, so it's easier to switch RPC providers if needed

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/603/commits/2c605026ced01da3a6379484abad317afc17c3fd